### DISCUSSION TITLE
Non-lexical Quotes

### DISCUSSION THREAD

**Discussion by ghost** [h3rald/min#148] [Ideas]
Accessing all symbols of upper scopes wouldn't good always. That may cause mistakenly using some predefined variables in upper scopes. So *non-lexical quotes* would help us. What do u think about? There may be rare cases of usage 😄  Looks useful i think.

**Comment by h3rald** [comment]
🤣 You gotta do better than that... show me a case where that would be really useful 😉

**Reply by ghost** [reply]
Hmm... i should think **a lot** for use cases 🤔 😃  ... but if something happens which related to this idea then we would discuss here.
